server/dex/dex.go

```diff
@@ -455,6 +455,7 @@ func NewDEX(cfg *DexConf) (*DEX, error) {
 		AuthManager: authMgr,
 		Markets:     marketTunnels,
 	})
+	startSubSys("OrderRouter", orderRouter)
```

Comment:
On to the server changes, `OrderRouter` is now a live subsystem as it has a latency queue to start and stop.